ONSdigital/eq-survey-runner
# Alter routing on qualification questions

### Context

This is the companion piece to the content changes on qualifications, which referred to later questions. This PR concerns bringing routing back into line with that guidance, re-ordering which qualification types are asked about.

The Trello card can be found here: https://trello.com/c/Vz8aIfOw/3165-routing-changes-qualifications-nisra

### How to test
Clone branch, run runner and launcher, try NISRA household or individual surveys. Progress through to qualifications, then see that it obeys the Trello card updates.

If you say no/none to **Degree**, **A-level**s, **GCSE**s and **NVQ**s, you should get asked about **Other** quals. If any or all are yes, you should skip it. In either case, the final question should be on **apprenticeships**.
